db/schema.rb

```diff
@@ -10,18 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161130153206) do
```

Comment:
Not sure why we are getting this change here. It seems that it is only changing the schema.